app/views/job_invocations/_tab_overview.html.erb

```diff
@@ -27,7 +27,12 @@
     <h5>
       <b><%= template_invocation.template.name %></b> <%= 'through' %> <%= _(RemoteExecutionProvider.provider_for(template_invocation.template.provider_type)) %>
     </h5>
-    <pre><%= InputTemplateRenderer.new(template_invocation.template, nil, template_invocation).preview %></pre>
+    <% target = template_invocation.targeting.hosts.first %>
```

Comment:
Based on the discussion here, https://github.com/theforeman/foreman_remote_execution/pull/26#discussion_r38207845, maybe we pick the first host with an OS?